src/main/java/com/github/sgov/server/controller/dto/VocabularyWorkspacesDto.java

```diff
@@ -0,0 +1,38 @@
+package com.github.sgov.server.controller.dto;
+
+import cz.cvut.kbss.jsonld.annotation.JsonLdAttributeOrder;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.Data;
+
+@Data
+@JsonLdAttributeOrder({"uri", "label", "basedOnVersion", "changeTrackingContext", "workspaces"})
+public class VocabularyWorkspacesDto extends VocabularyDto {
```

Comment:
I would call it VocabularyWithWorkspacesDto, otherwise the name is rather confusing for me -- seems like DTO is holder of workspaces not of a vocabulary.

I assume you are not following any convention from the project.